doc/howto/write_a_view/index.md

```diff
@@ -0,0 +1,429 @@
+# How to write a view {#howto_write_a_view}
+
+[TOC]
+
+This HowTo documents how to write a view using the standard library and some helpers from SeqAn3.
+
+\tutorial_head{Difficult, 120 min, \ref tutorial_concepts\,\ref tutorial_ranges, }
+
+\note
+Some of the links from this HowTo only resolve in the developer documentation, because
+they referred to entities from the seqan3::detail namespace.
+We recommend you open this tutorial from http://docs.seqan.de/seqan/3.0.0-master-dev/
```

Comment:
```suggestion
We recommend you to open this tutorial from [the developer documentation](http://docs.seqan.de/seqan/3.0.0-master-dev/)
```

Reply:
the second half of the suggestion yes, the first no :)
This is more polite.